### ISSUE TITLE
Unrecognized arguments

### ISSUE THREAD

**Issue by rvalenzuelar** [deeplycloudy/glmtools#27]
I'm using the following command:

`python make_GLM_grids.py -o /Users/raulvalenzuela/Google Drive/PythonProjects/GOES-R_GLM --fixed_grid --goes_position east --dx 10.0 --dy 10.0 --ctr_lon 0.0 --ctr_lat 0.0 OR_GLM-L2-LCFA_G16_s20190950200000_e20190950200200_c20190950200224.nc  OR_GLM-L2-LCFA_G16_s20190950200200_e20190950200400_c20190950200429.nc  OR_GLM-L2-LCFA_G16_s20190950200400_e20190950201000_c20190950201027`

but I'm getting this error message:

`
make_GLM_grids.py: error: unrecognized arguments:  OR_GLM-L2-LCFA_G16_s20190950200000_e20190950200200_c20190950200224.nc  OR_GLM-L2-LCFA_G16_s20190950200200_e20190950200400_c20190950200429.nc  OR_GLM-L2-LCFA_G16_s20190950200400_e20190950201000_c20190950201027
`
It also happens if I define the --start and --end keywords.

**Comment by deeplycloudy** [comment]
My guess is that the space in "Google Drive" is causing the problem. Does it work if you escape the space (Google\ Drive) or put the whole path in quotes?

**Comment by rvalenzuelar** [comment]
After using "Google\ Drive" I get:

```
Traceback (most recent call last):
  File "/Users/raulvalenzuela/miniconda2/envs/py36/lib/python3.6/site-packages/xarray/core/dataset.py", line 976, in _construct_dataarray
    variable = self._variables[name]
KeyError: 'group_id'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "make_GLM_grids.py", line 305, in <module>
    gridder(glm_filenames, start_time, end_time, **grid_kwargs)
  File "/Users/raulvalenzuela/Google Drive/PythonProjects/GOES-R_GLM/glmtools/grid/make_grids.py", line 826, in grid_GLM_flashes
    outputs = list(map(this_proc_each_grid, subgrids))
  File "/Users/raulvalenzuela/Google Drive/PythonProjects/GOES-R_GLM/glmtools/grid/make_grids.py", line 887, in proc_each_grid
    gridder.process_flashes(glm, **process_flash_kwargs_ij)
  File "/Users/raulvalenzuela/Google Drive/PythonProjects/GOES-R_GLM/glmtools/grid/make_grids.py", line 394, in process_flashes
    nadir_lon=nadir_lon)
  File "/Users/raulvalenzuela/Google Drive/PythonProjects/GOES-R_GLM/glmtools/io/mimic_lma.py", line 82, in read_flashes
    min_events=min_events, min_groups=min_groups)
  File "/Users/raulvalenzuela/Google Drive/PythonProjects/GOES-R_GLM/glmtools/io/glm.py", line 293, in subset_flashes
    return self.get_flashes(flash_ids)
  File "/Users/raulvalenzuela/Google Drive/PythonProjects/GOES-R_GLM/glmtools/io/glm.py", line 300, in get_flashes
    these_flashes = self.reduce_to_entities('flash_id', flash_ids)
  File "/Users/raulvalenzuela/Google Drive/PythonProjects/GOES-R_GLM/glmtools/io/traversal.py", line 217, in reduce_to_entities
    last_entity_ids = dataset[e_var].data
  File "/Users/raulvalenzuela/miniconda2/envs/py36/lib/python3.6/site-packages/xarray/core/dataset.py", line 1058, in __getitem__
    return self._construct_dataarray(key)
  File "/Users/raulvalenzuela/miniconda2/envs/py36/lib/python3.6/site-packages/xarray/core/dataset.py", line 979, in _construct_dataarray
    self._variables, name, self._level_coords, self.dims)
  File "/Users/raulvalenzuela/miniconda2/envs/py36/lib/python3.6/site-packages/xarray/core/dataset.py", line 80, in _get_virtual_variable
    ref_var = dim_var.to_index_variable().get_level_variable(ref_name)
  File "/Users/raulvalenzuela/miniconda2/envs/py36/lib/python3.6/site-packages/xarray/core/variable.py", line 1977, in get_level_variable
    raise ValueError("IndexVariable %r has no MultiIndex" % self.name)
ValueError: IndexVariable 'number_of_groups' has no MultiIndex
```

**Comment by deeplycloudy** [comment]
OK, looks like the first problem is fixed, and now you've run into #22. You can read the long description therein, but the solution is to pin `xarray` to version 0.10.9 (`conda install xarray=0.10.9`).

**Comment by rvalenzuelar** [comment]
Downgrading xarray to 0.10.9 solved the problem. Thanks.
